client-local/src/main/java/com/cloudera/livy/client/local/LocalClient.java

```diff
@@ -372,18 +369,30 @@ public void run() {
         String allJars = Joiner.on(",").join(livyJars, userJars);
         conf.set(SPARK_JARS_KEY, allJars);
       } else {
-        argv.add("--jars");
-        argv.add(livyJars);
+        argv.put("jars",livyJars);
       }
 
-      argv.add(jar);
-      argv.add("--remote-host");
-      argv.add(serverAddress);
-      argv.add("--remote-port");
-      argv.add(serverPort);
-
-      LOG.info("Running client driver with argv: {}", Joiner.on(" ").join(argv));
-      final Process child = new ProcessBuilder(argv.toArray(new String[argv.size()])).start();
+      //LOG.info("Running client driver with argv: {}", Joiner.on(" ").join(argv));
+      //-----Comment existing livy launcher
+     // final Process child = new ProcessBuilder(argv.toArray(new String[argv.size()])).start();
+        final SparkLauncher launcher= new SparkLauncher();
```

Comment:
Indentation here is incorrect. Also, missing space before `=`. And, for the following lines, see previous comment about the hash map you're creating. Just create the launcher first thing, and then set things directly in the launcher.